app/gui/view/src/project.rs

```diff
@@ -148,17 +147,11 @@ struct SearcherFrp {
 #[derive(Clone, CloneRef, Debug)]
 pub enum SearcherVariant {
     ComponentBrowser(component_browser::View),
-    /// We keep the old searcher in a Rc, as its memory size is much greater than the new one.
-    OldNodeSearcher(Rc<searcher::View>),
 }
 
 impl SearcherVariant {
     fn new(app: &Application) -> Self {
-        if ARGS.groups.feature_preview.options.new_component_browser.value {
-            Self::ComponentBrowser(app.new_view::<component_browser::View>())
-        } else {
-            Self::OldNodeSearcher(Rc::new(app.new_view::<searcher::View>()))
-        }
+        Self::ComponentBrowser(app.new_view::<component_browser::View>())
     }
 
     fn frp(&self, project_view_network: &frp::Network) -> SearcherFrp {
```

Comment:
If we went so far, we should also remove `SearcherVariant` altogether, and remove "feature-preview.new-component-browser" option from argument list (app/ide-desktop/lib/content-config/src/config.json)